corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -473,6 +482,32 @@
         {% endcompress %}
         {% endif %}
 
+
+        {% if request.use_maps or request.use_maps_async %}
+        {% compress js %}
+        <script type="text/javascript" src="{% static 'jquery-color/jquery.color.js' %}"></script>
+        <script type="text/javascript" src="{% static 'leaflet/dist/leaflet.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
+
+        {% if reqeust.maps_prefer_canvas %}
+        <script>L_PREFER_CANVAS = true;</script>
+        {% endif %}
+
+        {% if request.use_maps %}
+        {% compress js %}
+        <script src="{% static 'reports/javascripts/maps.js' %}"></script>
+        <script src="{% static 'reports/javascripts/maps_utils.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
+
+        {% if request.use_maps_async %}
+        {% compress js %}
+        <script src="{% static 'reports/javascripts/async_maps.js' %}"></script>
+        <script src="{% static 'reports/javascripts/maps_utils.js' %}"></script>
+        {% endcompress %}
+        {% endif %}
```

Comment:
Appologies - I had assumed this was already on prod. I'll kick the ticket back to interrupt to verify its fixed after deploy.